exercises/go-2-a/.meta/start_3.go

```diff
@@ -0,0 +1,38 @@
+package go2a
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var cutoffs = []struct {
+	amount      int
+	description string
+}{
+	{144, "gross"},
+	{20, "score"},
+	{12, "dozen"},
+}
+
+func appendDesc(description *strings.Builder, str string) {
+	if description.Len() != 0 {
+		description.WriteString(" or ")
+	}
+
+	description.WriteString(str)
+}
```

Comment:
Maybe move this one below `Describe`. The general rule in Go is to sort things by importance. Most important up top but still keep things that belong together. An exported function is definitely more important than a helper function.